src/catalyst-toolbox/catalyst-toolbox/src/bin/cli/snapshot/mod.rs

```diff
@@ -47,13 +50,14 @@ pub struct SnapshotCmd {
 impl SnapshotCmd {
     pub fn exec(self) -> Result<(), Report> {
         let raw_snapshot: RawSnapshot = serde_json::from_reader(File::open(&self.snapshot)?)?;
+        let dreps: Dreps = serde_json::from_reader(File::open(&self.dreps)?)?;
```

Comment:
the dreps file should be optional.  If its not set, it should default to an equivalent of:
```
{
  "dreps": []
}
```

Reply:
fixed